[2022-08-19 23:09:33,080] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T04:38:00+00:00 [queued]>
[2022-08-19 23:09:33,100] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T04:38:00+00:00 [queued]>
[2022-08-19 23:09:33,105] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 23:09:33,107] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-19 23:09:33,108] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 23:09:33,138] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_telegram_version> on 2020-01-01 04:38:00+00:00
[2022-08-19 23:09:33,152] {standard_task_runner.py:52} INFO - Started process 3303 to run task
[2022-08-19 23:09:33,178] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'sefa_dag', 'get_telegram_version', 'scheduled__2020-01-01T04:38:00+00:00', '--job-id', '585', '--raw', '--subdir', 'DAGS_FOLDER/sefa_dag.py', '--cfg-path', '/tmp/tmpl9tsbf76', '--error-file', '/tmp/tmpgruelm6h']
[2022-08-19 23:09:33,181] {standard_task_runner.py:80} INFO - Job 585: Subtask get_telegram_version
[2022-08-19 23:09:33,292] {task_command.py:371} INFO - Running <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T04:38:00+00:00 [running]> on host fd7a784b9251
[2022-08-19 23:09:33,431] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=sefa_dag
AIRFLOW_CTX_TASK_ID=get_telegram_version
AIRFLOW_CTX_EXECUTION_DATE=2020-01-01T04:38:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-01T04:38:00+00:00
[2022-08-19 23:09:33,434] {python.py:173} INFO - Done. Returned value was: 1.24.0
[2022-08-19 23:09:33,461] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=sefa_dag, task_id=get_telegram_version, execution_date=20200101T043800, start_date=20220819T230933, end_date=20220819T230933
[2022-08-19 23:09:33,532] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-19 23:09:33,587] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
